### DISCUSSION TITLE
请问一下这个是什么错，怎么解决

### DISCUSSION THREAD

**Discussion by justiceyuy** [qier222/YesPlayMusic#1251] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/48314460/150095159-b413b897-9cdf-40d1-a70b-495e25f56c7e.png)

**Comment by pan93412** [comment, marked as answer]
~~#1145~~ #1238 有臨時解決方案。

**Reply by justiceyuy** [reply]
咋临时解决？

**Reply by pan93412** [reply]
#1238 抱歉沒給直接連結
